### PR TITLE
[REF] auto-fix conventions

### PR DESCRIPTION
**Pull request by moylop260**

### CODE REVIEW

**Comment by eLBati** [comment]
`./sale_markup/tests/test_sale_markup.py:169:80: E501 line too long (100 > 79 characters)`

File: sale_floor_price/product.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 
 
 class Product(orm.Model):
+
     """Add floor price to product"""
```
Comment (bwrsandman) [inline sale_floor_price/product.py:28]: :-1:

File: sale_markup/tests/test_sale_markup.py [hunk 2]
```diff
@@ -78,6 +78,7 @@ def _trigger_on_changes(self, cr, uid, sale_order,
 
 
 class TestSaleMarkup(common.TransactionCase):
+
     """ Test the wizard for delivery carrier label generation """
```
Comment (bwrsandman) [inline sale_markup/tests/test_sale_markup.py:82]: :-1:

**Comment by yvaucher** [comment]
Superseeded by https://github.com/OCA/sale-financial/pull/12

**Pull request closed without merging**